Sadness, this is still failing:

```
Starting >>> warehouse_ros
  --- stderr: warehouse_ros
  CMake Error at /usr/share/cmake-3.16/Modules/CMakeTestCCompiler.cmake:60 (message):
    The C compiler
  
      "/usr/lib/ccache/cc"
  
    is not able to compile a simple test program.
  
    It fails with the following output:
  
      Change Dir: /root/target_ws/build/warehouse_ros/CMakeFiles/CMakeTmp
      
      Run Build Command(s):/usr/bin/make cmTC_af595/fast && /usr/bin/make -f CMakeFiles/cmTC_af595.dir/build.make CMakeFiles/cmTC_af595.dir/build
      make[1]: Entering directory '/root/target_ws/build/warehouse_ros/CMakeFiles/CMakeTmp'
      Building C object CMakeFiles/cmTC_af595.dir/testCCompiler.c.o
      /usr/lib/ccache/cc    -o CMakeFiles/cmTC_af595.dir/testCCompiler.c.o   -c /root/target_ws/build/warehouse_ros/CMakeFiles/CMakeTmp/testCCompiler.c
      ccache: error: Failed to create directory /home/runner/work/warehouse_ros/warehouse_ros//root/.ccache/tmp: Read-only file system
      make[1]: *** [CMakeFiles/cmTC_af595.dir/build.make:66: CMakeFiles/cmTC_af595.dir/testCCompiler.c.o] Error 1
      make[1]: Leaving directory '/root/target_ws/build/warehouse_ros/CMakeFiles/CMakeTmp'
      make: *** [Makefile:121: cmTC_af595/fast] Error 2
      
      
  
    
  
    CMake will not be able to correctly generate this project.
  Call Stack (most recent call first):
    CMakeLists.txt:2 (project)
```

--- a/.github/workflows/industrial_ci_action.yml
+++ b/.github/workflows/industrial_ci_action.yml
@@ -1,7 +1,7 @@
 # This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
 # For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
 
-name: CI
+name: BuildAndTest
 
 # This determines when this workflow is run
 on:
@@ -19,17 +19,8 @@ jobs:
         env:
           - {ROS_DISTRO: foxy, ROS_REPO: main}
           - {ROS_DISTRO: foxy, ROS_REPO: testing}
-    env:
-      CCACHE_DIR: $HOME/.ccache # Directory for ccache (and how we enable ccache in industrial_ci)
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      # This step will fetch/store the directory used by ccache before/after the ci run
-      - uses: actions/cache@v2
-        with:
-          path: ${{ env.CCACHE_DIR }}
-          key: ccache-${{ matrix.env.ROS_DISTRO }}-${{ matrix.env.ROS_REPO }}-${{ github.head_ref }}
-          restore-keys: ccache-${{ matrix.env.ROS_DISTRO }}-${{ matrix.env.ROS_REPO }}
-      # Run industrial_ci
       - uses: 'ros-industrial/industrial_ci@master'
         env: ${{ matrix.env }}

